installing/installing_ibm_z/installing-ibm-z.adoc

```diff
@@ -76,6 +81,12 @@ include::modules/installation-registry-storage-non-production.adoc[leveloffset=+
 
 include::modules/installation-complete-user-infra.adoc[leveloffset=+1]
 
+include::modules/installation-ibm-z-troubleshooting-and-debugging.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See also https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within OpenShift4 nodes without SSH].
```

Comment:
link:https://access.redhat.com/solutions/4387261[How to generate SOSREPORT within OpenShift4 nodes without SSH].